package/yast2.spec

```diff
@@ -139,9 +142,21 @@ mkdir -p %{buildroot}%{yast_schemadir}/control/rnc
 mkdir -p %{buildroot}%{yast_schemadir}/autoyast/rnc
 mkdir -p %{buildroot}%{_sysconfdir}/YaST2
 
+rake install DESTDIR="%{buildroot}"
+
+# symlink to call yast in ncurses
+mkdir -p %{buildroot}%{_sbindir}
+ln -snf yast2 %{buildroot}%{_sbindir}/yast
+# backward compatibility
+mkdir -p %{buildroot}/sbin
+ln -snf yast2 %{buildroot}/sbin/yast2
+ln -snf yast2 %{buildroot}/sbin/yast
+ln -snf yast2.8 %{buildroot}/usr/share/man/man8/yast.8
```

Comment:
We could avoid **all** these symlinks by using application specific links:
```sh
echo 'exec /usr/sbin/yast2'         > /sbin/yast2
echo 'exec -a yast /usr/sbin/yast2' > /sbin/yast
echo 'exec -a yast /usr/sbin/yast2' > /usr/sbin/yast
```

(the `-a` argument sets argv[0] so that yast2 knows to use ncurses)

`man` can also do it (although I can't find the reference docs for it)
```sh
echo '.so yast2.8' > /usr/share/man/man8/yast.8
```

I write them as `echo` but in fact we would have them in Git and install them to their places.

Reply:
OK, I try it and it complain ( but build pass ):

```
[   54s] yast2.x86_64: W: duplicate-executable yast ['/sbin/yast', '/usr/sbin/yast']
[   54s] yast2.x86_64: W: duplicate-executable yast2 ['/sbin/yast2', '/usr/sbin/yast2']
[   54s] This executable file exists in more than one standard binary directories. It
[   54s] can cause problems when dirs in $PATH are reordered.

[   54s] yast2.x86_64: W: script-without-shebang /sbin/yast
[   54s] yast2.x86_64: W: script-without-shebang /sbin/yast2
[   54s] yast2.x86_64: W: script-without-shebang /usr/sbin/yast
[   54s] This text file has executable bits set or is located in a path dedicated for
[   54s] executables, but lacks a shebang and cannot thus be executed.  If the file is
[   54s] meant to be an executable script, add the shebang, otherwise remove the
[   54s] executable bits or move the file elsewhere.
```

So looks like symlink is preferred over this exec stuff. BTW why you think the file is better?

Reply:
It's not really better, it just avoids symlinks, but in the end symlinks seem to be less trouble than this.